### DISCUSSION TITLE
Purpose of setting a field to both String and Text

### DISCUSSION THREAD

**Discussion by havardox** [dedupeio/dedupe#1154] [Q&A]
While browsing the [examples repo](https://github.com/dedupeio/dedupe-examples), I came across this line of code:

https://github.com/dedupeio/dedupe-examples/blob/master/record_linkage_example/record_linkage_example.py#L110-L111

I'm curious as to why the `title` field is set to both String and Text. How does it improve linking accuracy compared to using either String or Text alone?

**Comment by rderidder-lda** [comment]
https://docs.dedupe.io/en/latest/Variable-definition.html#multiple-variables-comparing-same-field

it simply allows for multiple methods to match on, rather than a single one
